README.md

```diff
@@ -31,7 +31,7 @@ You can install openomni in editable mode like this:
 ```
 git clone https://github.com/openaps/openomni.git
 cd openomni
-pip install -e .
+pip install -e . --process-dependency-links
```

Comment:
Is this necessary for rfcat?  Reading about the security implications and how they want to deprecate this feature, I think maybe it should be avoided.  We can list installation of rfcat as a separate step.

Reply:
For RFcat this is necessary if you want to install it with setup.py because it not registered on PiPy and we have a one line installer. 

I read the option has been put back in  because there is currently no other solution in the works:
https://github.com/pypa/pip/issues/4187

I would like to keep it in for now because it simplifies the install steps, but I will create PR tonight on RFCat with the needed changes to add rfcat to the PiPy registry. It's quite easy to add packages to pipy yourself.

The separate step was a bit confusing for me at the time to understand what RFCat is used for. I used the RFCat first as a CC compiler for the TI stick and then also for openomni as a python package. So when I found out it can be installed as a package, I liked this, because it helps in the logic of all the different steps and have a 1 install command (I started using Docker at work so I am really into 1 line installers now ;-))